apps/src/templates/codeReviewGroups/CodeReviewGroupsManager.jsx

```diff
@@ -152,11 +153,14 @@ const addDroppableIdToGroup = group => {
   if (group.unassigned) {
     group.droppableId = DROPPABLE_ID_UNASSIGNED;
   } else {
-    group.droppableId = `${DROPPABLE_ID_PREFIX}${group.id}`;
+    group.droppableId = getAssignedGroupDroppableId(group.id);
   }
   return group;
 };
 
+// Exported for a test
+export const getAssignedGroupDroppableId = id => `${DROPPABLE_ID_PREFIX}${id}`;
```

Comment:
readability nit: it's clearer to use string concatenation here than to use a template with string interpolation (`DROPPABLE_ID_PREFIX + id`)